Formula/quictls.rb

```diff
@@ -0,0 +1,132 @@
+class Quictls < Formula
+  desc "This is a fork of OpenSSL to enable QUIC"
+  homepage "https://github.com/quictls/openssl"
+  url "https://github.com/quictls/openssl/archive/refs/tags/openssl-3.0.9-quic1.tar.gz"
+  version "3.0.9"
+  sha256 "7b76b0968642a3929fabc96acd28302f631d339a90a91fd434cde425a3ec6838"
+  license "Apache-2.0"
+
+  keg_only :shadowed_by_macos, "macOS provides LibreSSL"
+
+  depends_on "ca-certificates"
+
+  on_linux do
+    keg_only "it conflicts with the `openssl@3` formula"
+
```

Comment:
```suggestion
  keg_only "it conflicts with OpenSSL"

  depends_on "ca-certificates"

  on_linux do
```
Let's just have on `keg_only` reason here.